lib/shopify_app/version.rb

```diff
@@ -1,3 +1,3 @@
 module ShopifyApp
-  VERSION = '9.0.4'.freeze
+  VERSION = '10.0.0'.freeze
```

Comment:
can you make sure this is changed in package.json too, per the [release docs](https://github.com/Shopify/shopify_app/blob/master/docs/Releasing.md)